### PR TITLE
new content + fixes

### PR DESCRIPTION
**Pull request by mairaw**

### CODE REVIEW

File: xml/Microsoft.JScript.Vsa/VsaEngine.xml [hunk 1]
```diff
@@ -68,8 +68,9 @@
       </Attributes>
       <Parameters />
       <Docs>
-        <summary>This class, member, or property is internal to the script engine and should not be called from your code.</summary>
+        <summary>Initializes a new instance of the <see cref="T:Microsoft.JScript.Vsa.VsaEngine" />  class.</summary>
```
Comment (gewarren) [inline xml/Microsoft.JScript.Vsa/VsaEngine.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:Microsoft.JScript.Vsa.VsaEngine" /> class.</summary>
```

File: xml/Microsoft.JScript.Vsa/VsaEngine.xml [hunk 2]
```diff
@@ -90,9 +91,11 @@
         <Parameter Name="fast" Type="System.Boolean" />
       </Parameters>
       <Docs>
-        <param name="fast">To be added.</param>
-        <summary>This class, member, or property is internal to the script engine and should not be called from your code.</summary>
+        <param name="fast">
+          <see langword="true" /> to indicate fast mode; otherwise, <see langword="false" />.</param>
+        <summary>Initializes a new instance of the <see cref="T:Microsoft.JScript.Vsa.VsaEngine" />  class, and optionally sets fast mode.</summary>
```
Comment (gewarren) [inline xml/Microsoft.JScript.Vsa/VsaEngine.xml]:
```suggestion
        <summary>Initializes a new instance of the <see cref="T:Microsoft.JScript.Vsa.VsaEngine" /> class and optionally sets fast mode.</summary>
```

File: xml/Microsoft.JScript.Vsa/VsaEngine.xml [hunk 3]
```diff
@@ -324,9 +332,9 @@
         <Parameter Name="callingTypeHandle" Type="System.RuntimeTypeHandle" />
       </Parameters>
       <Docs>
-        <param name="callingTypeHandle">To be added.</param>
-        <summary>This class, member, or property is internal to the script engine and should not be called from your code.</summary>
-        <returns>To be added.</returns>
+        <param name="callingTypeHandle">The calling type handle.</param>
+        <summary>Creates the engine with type.</summary>
```
Comment (gewarren) [inline xml/Microsoft.JScript.Vsa/VsaEngine.xml]: Is there a word missing here?

Comment (mairaw) [reply xml/Microsoft.JScript.Vsa/VsaEngine.xml]: changed it.

**Pull request merged**